ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -146,23 +152,47 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar })
     return function (error) {
       updateProgress({ showProgress : false });
 
-      enqueueSnackbar(`${msg}: ${error}`, { variant : "error",
+      enqueueSnackbar(`${msg}: ${error}`, {
+        variant : "error",
         action : function Action(key) {
           return (
             <IconButton key="close" aria-label="Close" color="inherit" onClick={() => closeSnackbar(key)}>
               <CloseIcon />
             </IconButton>
           );
         },
-        autoHideDuration : 8000, });
+        autoHideDuration : 8000,
+      });
     };
   }
 
   return (
     <>
       <div style={{ padding : "0.5rem" }}>
-        <div style={{ margin : "0 0 2rem auto", width : "fit-content" }}>
-          <ViewSwitch view={viewType} changeView={setViewType} />
+        <div style={{ margin : "0 0 2rem auto", display : "flex", justifyContent : "space-between", paddingLeft : "1rem" }}>

```

Comment:
Can we extract the style statement, since there are lots of styles being applied here and the char count is I guess surpassing 80.